test/GroupByTest.cpp

```diff
@@ -674,4 +676,69 @@ TEST_F(GroupBySpecialCount, computeGroupByForFullIndexScan) {
   // TODO<joka921> Add a test with only one column
 }
 
+namespace {
+// A helper function to set up expression trees in the following test.
+template <typename ExprT>
+auto make = [](auto&&... args) -> SparqlExpression::Ptr {
+  return std::make_unique<ExprT>(AD_FWD(args)...);
+};
+}  // namespace
+TEST(GroupBy, GroupedVariableInExpressions) {
```

Comment:
Add a new line before this line and move the "Test the following SPARQL query: ..." comment from inside the test to before this test (because it really describes that the whole test does, not just a part of it).